packages/db-states-redis/src/lib/states/statesInRedisClient.ts

```diff
@@ -1224,6 +1224,7 @@ export class StateRedisClient {
             throw new Error(tools.ERRORS.ERROR_DB_CLOSED);
         }
 
+        // @ts-expect-error very strange error
```

Comment:
... I think you have installed an old version of ``@iobroker/types`` globally again